src/node_zlib.cc

```diff
@@ -608,7 +608,8 @@ class CompressionStream : public AsyncWrap, public ThreadPoolWork {
   }
 
   static void* AllocForBrotli(void* data, size_t size) {
-    size += sizeof(size_t);
+    constexpr size_t offset = std::max(sizeof(size_t), alignof(max_align_t));
```

Comment:
If you move this out of the function duplication could be avoided.

Reply:
As already mentioned in the commit message, this prevents a crash of Node on OE, it is not intended as a code cleanup. I would prefer to keep it as simple as possible so it can be backported.